### PR TITLE
Add aruco and d4345 models and attached them to drone and rover

### PR DESCRIPTION
**Pull request by VaibhavSanjay**
Added the Aruco marker and the D435 realsense camera to the models directory. Then modified the iris and r1_rover sdf to include those models.

### CODE REVIEW

File: models/iris/iris.sdf.jinja [hunk 1]
```diff
@@ -418,6 +418,15 @@
...
     </plugin>
+    <include>
+      <uri>model://d435</uri>
+      <pose>0 0 0 0 1.57 0</pose>
+      <name>d435</name>
+    </include>
+    <joint name="d435_joint" type="fixed">
+      <child>d435::link</child>
+      <parent>base_link</parent>
+    </joint>
```
Comment (Jaeyoung-Lim) [inline models/iris/iris.sdf.jinja:429]: I don't think we want the d435 model added to the iris by default.

File: models/r1_rover/r1_rover.sdf.jinja [hunk 2]
```diff
@@ -515,6 +515,15 @@
         <use_parent_model_frame>1</use_parent_model_frame>
       </axis>
     </joint>
+    <include>
+      <uri>model://aruco_marker</uri>
+      <pose>0 0 0 0 0 0</pose>
+      <name>aruco</name>
+    </include>
```
Comment (Jaeyoung-Lim) [inline models/r1_rover/r1_rover.sdf.jinja:522]: I don't think we want the aruco marker added to the r1 rover by default, but probably we can add an additional model that supports it

**Review by Jaeyoung-Lim — CHANGES_REQUESTED** [review]
@VaibhavSanjay Thanks for the contribution

I think this could be a useful addition, but could you explain why the use case? Since PX4 does not do any functional testing or supports any functionalities related to the Aruco markers, I am not sure how we want to use them.

Also, I don't think we want the aruco markers added to the default model since it is used for CI testing and we don't want any camera models in there.

**Comment by VaibhavSanjay** [comment]
Hi @Jaeyoung-Lim !

So, I'm kinda new to Github, and I accidentally made a pull request on this fork when I really just wanted to make a modification for my own use (oops). Now that you mention it, I was trying to make a separate model for these modifications but I couldn't actually figure out how to do it, so I just modified the main stuff and it worked. Sorry about that again, but thank you for the comment!

**Pull request closed without merging**